src/index.html

```diff
@@ -1,17 +1,172 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset class="fieldset person-info">
+        <legend>Personal information</legend>
+        <label class="form-field surname">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+          >
+        </label>
+
+        <label class="form-field name">
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+          >
+        </label>
+
+        <label class="form-field how-old">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field date-birth">
+          Full date of birth:
+          <input
+            type="date"
+            name="trip-start"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input 
+          type="checkbox" 
+          name="term"
+          required
```

Comment:
please fix the spaces
```suggestion
            type="checkbox" 
            name="term"
            required
```